pinot-query-planner/src/main/java/org/apache/calcite/sql/fun/PinotOperatorTable.java

```diff
@@ -207,6 +222,49 @@ public final void initNoDuplicate() {
     }
   }
 
+  /**
+   * This registers transform functions defined in {@link org.apache.pinot.common.function.TransformFunctionType}
+   * which are multistage enabled.
+   */
+  private void initTransformFunctions() {
+    // Walk through all the Pinot transform types and register those that are supported in multistage.
+    for (TransformFunctionType transformFunctionType : TransformFunctionType.values()) {
+      if (transformFunctionType.getSqlKind() == null) {
+        // Skip registering functions which are standard Calcite functions and functions which are not yet supported.
+        continue;
+      }
+
+      // Register the aggregation function with Calcite along with all alternative names
+      List<PinotSqlTransformFunction> sqlTransformFunctions = new ArrayList<>();
+      PinotSqlTransformFunction aggFunction = generatePinotSqlTransformFunction(transformFunctionType.getName(),
+          transformFunctionType, false);
+      sqlTransformFunctions.add(aggFunction);
+      List<String> alternativeFunctionNames = transformFunctionType.getAliases();
+      if (alternativeFunctionNames == null || alternativeFunctionNames.size() == 0) {
+        // If no alternative function names are specified, generate one which converts camel case to have underscores
+        // as delimiters instead. E.g. boolAnd -> BOOL_AND
+        String alternativeFunctionName =
+            convertCamelCaseToUseUnderscores(transformFunctionType.getName());
```

Comment:
this convert should be moved to transform function type. this is the way we going to handle `_` separated vs camel case unless specific naming is specified.

Reply:
removed.